cmd/monaco/integrationtest/cleanup.go

```diff
@@ -82,8 +82,8 @@ func cleanupSettings(t *testing.T, fs afero.Fs, manifestPath string, loadedManif
 		}
 		for _, configs := range cfgsForEnv {
 			for _, cfg := range configs {
-				if typ, ok := cfg.Type.(*config.SettingsType); ok {
-					extID := idutils.GenerateExternalID(typ.SchemaId, cfg.Coordinate.ConfigId)
+				if typ, ok := cfg.Type.(config.SettingsType); ok {
+					extID, _ := idutils.GenerateExternalID(cfg.Coordinate)
```

Comment:
does it even make sense to continue with delete if the extId generation failed? Before we panicked. 
Maybe we should continue rather than even attempt the delete?